toplevel/coqtop_opt_bin.ml

```diff
@@ -1,3 +1,16 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *   INRIA, CNRS and contributors - Copyright 1999-2018       *)
+(* <O___,, *       (see CREDITS file for the list of authors)           *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
 let drop_setup () = Mltop.remove ()
 
-let _ = drop_setup ()
+(* Main coqtop initialization *)
+let _ =
+  drop_setup ();
```

Comment:
Is this drop_setup really useful for native?

Reply:
This was the old behavior so I just preserved it [for now]